hazelcast/src/test/java/com/hazelcast/cluster/AdvancedClusterStateTest.java

```diff
@@ -422,7 +422,7 @@ private void changeClusterStateEventually(HazelcastInstance hz, ClusterState new
         final Cluster cluster = hz.getCluster();
```

Comment:
Is there a test which can reproduce the original issue? So that we know your fix really solves this issue.